app/helpers/application_helper.rb

```diff
@@ -32,6 +32,13 @@ def link_to(*args, &block)
     end
   end
 
+  def link_to_function(name, function, html_options = {})
```

Comment:
Unfortunately `link_to_function` was removed from Rails on Rails 4, but we use it extensively.